test/acceptance/test/add_tests.go

```diff
@@ -283,11 +283,14 @@ var _ = Describe("Weave GitOps Add App Tests", func() {
 		private := false
 		tip := generateTestInputs()
 		appName := tip.appRepoName
-		appRepoRemoteURL := "ssh://git@gitlab.com/" + gitlabPublicGroup + "/" + tip.appRepoName + ".git"
+		configRepoName := "config-" + appName
 
-		addCommand := "add app . --auto-merge=true"
+		configRepoURL := "ssh://git@gitlab.com/" + gitlabPublicGroup + "/" + configRepoName + ".git"
```

Comment:
I would suggest to use `fmt.Sprintf` to build this string for readability

Reply:
I agree, but I decided t keep it this way because that's how URLs are being built across the whole acceptance suite, I'd rather keep consistency over something I prefer.